crates/nargo/src/toml.rs

```diff
@@ -5,15 +5,16 @@ use std::path::Path;
 use crate::errors::CliError;
 
 #[derive(Debug, Deserialize, Clone)]
-pub struct Config {
-    pub package: Package,
-    pub dependencies: BTreeMap<String, Dependency>,
+pub(crate) struct Config {
+    #[allow(dead_code)]
```

Comment:
why is the dead_code lint needed?

Reply:
We never actually read the `package` field anywhere. This isn't currently picked up as it's listed as public (despite not being in the crate interface) but when we restrict its visibility to the crate then rust can see that we don't use it.

Reply:
Ahh , okay I think thats a separate issue that we need to look into in the future, re where are the dead_code lints